### DISCUSSION TITLE
rshell and python 3.10

### DISCUSSION THREAD

**Discussion by smhodge42** [micropython/micropython#9665] [General]
I just rebuilt my Windows10 system and installed the latest python version, 3.10, as well as rshell. I get an error message with rshell:

AttributeError: module 'collections' has no attribute 'Callable' 

Is this correct, that rshell does not work with python 3.10? If so, is there a work-around?  I hope I don't have to go back to an earlier version of python. I did not have any issues with python 3.7.

**Comment by dlech** [comment]
Sounds like https://github.com/dhylands/rshell/issues/180

**Reply by smhodge42** [reply]
Thanks. Like one of the other posts suggested, simply installing pyreadline3 was all I needed to do.
